extensions/ots/message_configuration/templates/messaging_help.html

```diff
@@ -0,0 +1,53 @@
+{% load i18n %}
+
+<p>
+  {% trans "Here you can configure message templates for different messages hypha sends out.  You can configure the footer and header (which will apepar on each email), as we well as configure whether hypha sends out messages by default." %}
+</p>
+<p>
+  {% trans "Click 'Add' at the bottom to configure a template for an outgoing message type." %}
+</p>
+<p>
+  {% trans "Variables that will get replaced follow.  Use the uppercase name (before the colon) in any subject, email message, or slack message." %}
+</p>
+
+<p>{% trans "When the submission is available, the following are available:" %}</p>
+
+<ul>
```

Comment:
Add class `w-mb-4` to add some margins to the lists.

Reply:
Good call!  I'll make the change depending on how the overarching PR discussion goes :)